### DISCUSSION TITLE
cURL Daily Builds for Arch Linux and Arch Based Systems

### DISCUSSION THREAD

**Discussion by pheiduck** [curl/curl#10841] [Show and tell]
I would like to announce that I have been building a curl-daily package for Arch Linux (and Linux distributions based on it) for some time now. The PKGBUILD is pretty the same as that from Arch Linux: https://gitlab.archlinux.org/archlinux/packaging/packages/curl/-/blob/main/PKGBUILD

It uses curl daily snapshots: https://curl.se/snapshots/
and builds a compatible package for Arch Linux.

So hey, if there are Arch Linux users / developers here is the [PKGBUILD](https://raw.githubusercontent.com/pheiduck/curl-daily/main/PKGBUILD) for it. You can also use the pre-builds (automated builds): https://github.com/pheiduck/curl-daily/releases

A gift for the curl community and developers ❤️

**Comment by bagder** [comment]
Nice! Do you think we should add a link to these like we link to the windows snapshots on https://curl.se/snapshots/ ?

**Reply by bagder** [reply]
I added a link, now visible on https://curl.se/snapshots/

**Reply by pheiduck** [reply]
Thank you @bagder!
